runtime/metricsview/executor.go

```diff
@@ -184,6 +184,10 @@ func (e *Executor) Query(ctx context.Context, qry *Query, executionTime *time.Ti
 		return nil, err
 	}
 
+	if err := e.rewritePercentOfTotals(ctx, qry, e.metricsView, e.security); err != nil {
```

Comment:
This rewrite should be moved up above the `rewriteQueryDruidExactify` rewrite (since I think the exactify rewrite may try computing the measure)